app/main.py

```diff
@@ -1 +1,19 @@
-# write your code here
+from app.errors import VaccineError, NotWearingMaskError
+from app.cafe import Cafe
+
+
+def go_to_cafe(friends: list, cafe: Cafe) -> str:
+    all_vaccinated = True
+    masks_to_buy = 0
+    for friend in friends:
+        try:
+            cafe.visit_cafe(friend)
+        except VaccineError:
+            all_vaccinated = False
```

Comment:
There is no need for the `all_vaccinated` flag, in case even one person is not vaccinated you could return the required string straight away. Remove this flag